.github/workflows/release.yml

```
@@ -0,0 +1,76 @@
+        with:
+          name: ${{ matrix.artifact_name }}
+          path: ${{ matrix.artifact_name }}
+
+  publish:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v2
```

actionlint (1.7.4)

the runner of "actions/download-artifact@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)